apps/service_providers/tests/test_assistant_runnable.py

```diff
@@ -590,5 +590,6 @@ def _create_run(
         status=status,
         thread_id=thread_id,
         tools=[],
+        parallel_tool_calls=False,
```

Comment:
can you say more about this change

Reply:
I think [this link](https://platform.openai.com/docs/guides/function-calling/configuring-parallel-function-calling) explains it best. Basically do we want the model to output multiple function calls, or one at a time. This is a new required param, and since this is a test, the value of this param doesn't really matter